metrics/ci_integrations/docs/02_ci_integration_user_guide.md

```diff
@@ -134,9 +135,73 @@ destination:
 
 _**Note**: The order of source and destination parts doesn't matter - the first part can be `destination` and the second can be `source`. But both parts are required and must be presented in the configuration part._
 
-Well done! Your configuration file is ready to use within the `ci_integrations sync` command. The [next section](#making-things-work) describes
- how
- to make all things work together!
+Well done! Your configuration file is ready to use within the `ci_integrations sync` command. The [next section](#validating-configuration-file) describes how to validate the created configuration file.
+
+### Validating configuration file
+
+The CI Integrations tool provides functionality to validate the configuration files to ensure that all fields of the configuration file are correct. To do that, use the `validate` command of the CI Integrations tool. Consider the following example that demonstrates how to use the `validate` command:
+
+``` bash
+ci_integrations validate --config-file="path/to/config_file.yaml"
+```
+
+_**Note**: This step is optional. You can use the `sync` command without the configuration file validation, however, the `sync` command does not provide the detailed failure reasons, so it would be hard to figure out what went wrong._
+
+#### Validate command output
+
+The `validate` command provides a detailed output on each configuration file's field. The output for the whole configuration file is called a `validation result`. The `validation result`, in its turn, consists of `field validation results` that represent a validation result for a specific configuration file's field.
+
+Each `field validation result` holds the following information:
+- A configuration file field's name.
+- A validation conclusion for the field. Each conclusion has its specific visual marker that represents this conclusion. 
+
+Consider the following table that describes all existing conclusions and their meanings:
+
+| Conclusion | Marker | Meaning |
+| --- | --- | --- | 
+| `valid` | `[+]` | The field is valid. | 
+| `invalid` | `[-]` | The field is invalid. |
+| `unknown` | `[?]` | The field can't be validated due to some reasons or the clear validation conclusion can't be provided. |
+
+- An additional context that contains any related information on this configuration field's validation, e.g. an access token does not have the required permission to validate a specific field. This part of a `field validation result` is optional, so it can be absent.
```

Comment:
```suggestion
- An additional context that contains any related information on this configuration field's validation, e.g. an access token does not have the required permission to validate a specific field. This part of a `field validation result` is optional so, it can be absent.
```